index.html

```diff
@@ -0,0 +1,203 @@
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet"  href="styles.css">
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <a class="logo" href="#">Tiago L. Marin</a>
+            <div class="menu">
+                <img src="/assets/icon-menu.png" alt="Menu Button">
+            </div>
+        </header>
+            <section class="intro">
+                <h1>Hey There.<br>I'm Tiago</h1>
+                <h2>I'm a Software Developer</h2>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+                </p>
+                <ul class="social-icons">
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-gitHub.png" alt="Github Page">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-linkedin.png" alt="Linked-in  Page">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-angelist.png" alt="Github Page">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-twitter.png" alt="Twitter">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#" target="_blank">
+                            <img src="/assets/icon-medium.png" alt="Github Page">
+                        </a>
+                    </li>
+                </ul>
+            </section>
+            <section class="projects-grid">
+                <div class="grid-title">
+                    <h3>My Recent Works</h3>   
+                </div>
+                <article class="featured">
+                    <img src="/assets/project-featured.jpg" alt="Menu Button">                        
+                    <div class="wrap-card">
```

Comment:
- [ ] [REQUIRED] Your  `project-featured.jpg` picture is not display on work section. Please make changes!

![image](https://user-images.githubusercontent.com/63915024/170846360-ce7e49fd-88a7-4563-a142-3b5e5dbffe84.png)